go.mod

```diff
@@ -29,11 +29,15 @@ require (
 )
 
 require (
+	github.com/Masterminds/sprig/v3 v3.2.3
```

Comment:
@zhujian7 @zhiweiyin318  This `sprig` repo is also depended by the `helm` repo, it's provide funcs like `with` and `nindent`.